### DISCUSSION TITLE
What formatter is used in RecBole?

### DISCUSSION THREAD

**Discussion by ChenglongMa** [RUCAIBox/RecBole#1874] [Q&A] [answered]
Hi Team,

May I know the Python formatter used in this project?
I noticed a `style.cfg` exists in the root folder, which is for Yapf. And `Black` is used in GitHub workflow for lint check.

I am asking because I want to use the same settings as yours. Then the git will only highlight my changes and not include format changes.

Thanks!

**Comment by zhengbw0324** [comment, marked as answer]
@ChenglongMa 
Hello, you can refer to our modifications in [PR-1349](https://github.com/RUCAIBox/RecBole/pull/1349)

**Reply by ChenglongMa** [reply]
Great! Thanks @zhengbw0324.
